### DISCUSSION TITLE
New to DiSCoVeR, trying adaptive design

### DISCUSSION THREAD

**Discussion by jamesmalmeida** [sparks-baird/mat_discover#105] [Q&A]
Dear developer,

Thanks for the great code!
I am trying to use DiSCoVeR with a dataset I generated, although I am struggling with the suggest experiment routine.
It seems the suggestions are only from my dataset, not for new compositions, is this behavior the expected one, or I am missing something?

Best regards,
James

**Comment by sgbaird** [comment]
@jamesmalmeida thanks! Happy to help here.

DiSCoVeR assumes a predefined list of candidates rather than searching in a continuous sense (latter not implemented). There are a few options for predefined lists, and the choice depends on your application though. For example, whether you include only experimentally verified compounds in your search or allow theoretical compounds. Whether you want to use a smaller, more tightly curated dataset (Materials Project) vs. a larger, more exploratory dataset (NOMAD).

- https://matsci.org/t/extract-chemical-formulas-stability-measure-identifier-from-all-nomad-entries-excluding-certain-periodic-elements/39670/5
- example using the NOMAD dataset https://github.com/cseeg/DiSCoVeR-SuperCon-NOMAD-SMACT/blob/main/main.ipynb

Do you mind describing more about your project?

Also, how many training datapoints do you have?

Sterling

**Reply by jamesmalmeida** [reply]
@sgbaird, thanks for the very fast reply!

I think I understood now. I am trying to use a dataset I extracted with Natural Language Processing, with different mechanical properties.
I used this dataset to train the Discover routine, and generated a val_df with the same compositions but zeros on the target.

If I understood correctly, I should add new compositions to the val_df that are not in my original dataset, so them different experiments can be suggested based on the prediction of the model? Or do I need to load the NOMAD or Materials Project database anyway?

Thanks again!
James

**Comment by sgbaird** [comment]
That's correct that you just need new compositions in `val_df`. Whether you use NOMAD, MP, another database, or some list that you generate on your own is your design decision. Great questions and clarifications!

**Reply by jamesmalmeida** [reply]
Thanks a lot @sgbaird, I will do it this way.
